src/map/atcommand.h

```diff
@@ -79,8 +79,10 @@ struct AtCommandInfo {
 struct atcmd_binding_data {
 	char command[ATCOMMAND_LENGTH];
 	char npc_event[ATCOMMAND_LENGTH];
-	int group_lv;
-	int group_lv_char;
+	int group_lv; // DEPRECATED
+	int group_lv_char; // DEPRECATED
+	char *at_groups; // quick @commands "can-use" lookup
```

Comment:
Why is this a `char *` if it stores an array of `bool`? (side note: to avoid ambiguity, this should be a VECTOR, so that it self-documents that it's a vector rather than a pointer to a single entry)